### PR TITLE
Coverage improved.

### PR DESCRIPTION
**Pull request by uweschaefer**
please review

### CODE REVIEW

File: factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java [hunk 1]
```diff
@@ -30,31 +30,32 @@ public JsonParamConverterProvider(@NonNull ObjectMapper objectMapper) {
...
-            } catch (IOException e) {
+            } catch (Exception e) {
                 throw new BadRequestException();
             }
         }
 
         @Override
         @SneakyThrows
-        public String toString(T value) {
+        public String toString(@NonNull T value) {
```
Comment (JoergAdler) [inline factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java]: here, no nun null!

File: factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java [hunk 2]
```diff
@@ -30,31 +30,32 @@ public JsonParamConverterProvider(@NonNull ObjectMapper objectMapper) {
...
             return objectMapper.writeValueAsString(value);
         }
 
     }
 
     @Override
-    public <T> ParamConverter<T> getConverter(Class<T> rawType, Type genericType,
-            Annotation[] annotations) {
+    public <T> ParamConverter<T> getConverter(@NonNull Class<T> rawType, @NonNull Type genericType,
+            @NonNull Annotation[] annotations) {
```
Comment (JoergAdler) [inline factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java]: Here no nun null!

File: factcast-server-rest/src/main/java/org/factcast/server/rest/resources/FactSpecParam.java [hunk 3]
```diff
@@ -6,22 +6,30 @@
```
Comment (JoergAdler) [inline factcast-server-rest/src/main/java/org/factcast/server/rest/resources/FactSpecParam.java]: delete this (the whole file)

File: factcast-server-rest/src/test/java/org/factcast/server/rest/resources/FactSpecParam0Test.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package org.factcast.server.rest.resources;
```
Comment (JoergAdler) [inline factcast-server-rest/src/test/java/org/factcast/server/rest/resources/FactSpecParam0Test.java]: delete this (the whole file)

File: factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java (hunk 1 above)
Comment (uweschaefer) [inline factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java]: crap...

File: factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java (hunk 2 above)
Comment (uweschaefer) [inline factcast-server-rest/src/main/java/org/factcast/server/rest/resources/converter/JsonParamConverterProvider.java]: ok

File: factcast-server-rest/src/main/java/org/factcast/server/rest/resources/FactSpecParam.java (hunk 3 above)
Comment (uweschaefer) [inline factcast-server-rest/src/main/java/org/factcast/server/rest/resources/FactSpecParam.java]: yep, removed it

File: factcast-server-rest/src/test/java/org/factcast/server/rest/resources/FactSpecParam0Test.java (hunk 4 above)
Comment (uweschaefer) [inline factcast-server-rest/src/test/java/org/factcast/server/rest/resources/FactSpecParam0Test.java]: removed

**Pull request merged**